tests/imports.py

```diff
@@ -30,15 +30,18 @@ def test_top_level(self):
         current_structure = _strip_magic_module_attributes(ls=neuroconv.__dict__)
         expected_structure = [
             # Sub-modules
+            "basedatainterface",
+            "basetemporalalignmentinterface",
+            "baseextractorinterface",
             "nwbconverter",
             "tools",  # Attached to namespace by NWBConverter import
             "utils",  # Attached to namespace by NWBconverter import
             # Exposed attributes
             "NWBConverter",
             "ConverterPipe",
             "BaseDataInterface",
-            "TemporallyAlignedDataInterface",
-            "basedatainterface",  # Imported by ConverterPipe as a type-hint
```

Comment:
I think it's from intentional importing not just type-hints now, since I did adjust the core `__init__` to expose these 3 base data interfaces

So when the BaseDataInterface gets imported by one of the submodules (the all lower-case ones) the submodule itself gets set in the module `__dict__` as well